server/version.go

```diff
@@ -1,29 +1,27 @@
 package server
 
 import (
+	"github.com/kubernetes-incubator/cri-o/version"
 	"golang.org/x/net/context"
 	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
 )
 
-// Version returns the runtime name, runtime version and runtime API version
-func (s *Server) Version(ctx context.Context, req *pb.VersionRequest) (*pb.VersionResponse, error) {
-
-	runtimeVersion, err := s.Runtime().Version()
-	if err != nil {
-		return nil, err
-	}
-
+const (
+	// kubeAPIVersion is the api version of kubernetes.
 	// TODO: Track upstream code. For now it expects 0.1.0
-	version := "0.1.0"
-
-	// taking const address
-	rav := runtimeAPIVersion
-	runtimeName := s.Runtime().Name()
+	kubeAPIVersion = "0.1.0"
+	// containerName is the name prepended in kubectl describe->Container ID:
+	// cri-o://<CONTAINER_ID>
+	containerName     = "cri-o"
```

Comment:
I think we decided that the branding has gone to "CRI-O" should we all caps this too?  I don't think so, but thought I'd throw the question out there just in case.

Reply:
Lowercase, this is user facing in kube, no need for uppercase at all

Reply:
👍 Thanks for the confirmation.